src/components/Publish/Pricing/Dynamic.tsx

```diff
@@ -78,6 +78,9 @@ export default function Dynamic({ content }: { content: any }): ReactElement {
           name="amountOcean"
           datatokenOptions={{ symbol: 'OCEAN', name: 'Ocean Token' }}
           weight={`${Number(weightOnOcean) * 10}%`}
+          min={
+            price > 0 ? (price * Number(weightOnOcean) * 10).toString() : '50'
```

Comment:
no magical numbers please, where is the `50` coming from? Would assume this refers to the defaults we set, and they are set in `Publish/_constants.ts`. You can get the defaults easily via `initialValues` from the Formik context.

Reply:
also please only use Decimal function to do the math . Also use `.toDecimalPlaces(MAX_DECIMALS)` at the end.

Reply:
I took the `50` from the yup validation schema where is also magically set, so I guess there should be a constant added for min values inside `_constants.ts` and that to be used in multiple places